src/app/resources/resources-add.component.html

```diff
@@ -0,0 +1,88 @@
+<h2 i18n>New Resource</h2>
+<form [formGroup]="resourceForm" (ngSubmit)="onSubmit(value)" novalidate>
```

Comment:
does this parameter is required in onSubmit function as you have called  this.addCourse(this.resourceForm.value) to take value of resource form.